### ISSUE TITLE
Content gets cutoff when Text spacing is applied violating WCAG 2.1 Guidelines.

### ISSUE THREAD

**Issue by RakshithDattatraya** [microsoft/vscode#230771]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.94
- OS Version: Tested on VScode.dev 

Steps to Reproduce:

1. Enter large text in vscode.dev.
2. Modify text spacing using this bookmark https://codepen.io/stevef/full/YLMqbo /tool/manually. The content in the editor get cutoff thereby failing [WCAG Guidelines](https://www.w3.org/TR/WCAG21/#text-spacing) for accessiblity
![image](https://github.com/user-attachments/assets/67d0082b-4e83-45d6-a81b-bf942fd2cdb7)
*This issue also persists in Monaco Editor.

**Comment by RakshithDattatraya** [comment]
@alexdima Could you please help me with this issue.
